docs/modules/reference/pages/telemetryd/protocols/openconfig.adoc

```diff
@@ -115,7 +122,7 @@ Parameters are passed to the connector and are interpolated for node/interface a
 | none
 
 | frequency
-| Frequency at which OpenConfig data can be streamed.
+| Frequency at which OpenConfig data can be streamed. This is parsed as milliseconds(ms) for jti and nanoseconds(ns) for gnmi
```

Comment:
Can we unify this? Always make it ms or ns and let the impl below do the conversions?

Reply:
Historically speaking, I would have leaned toward using milliseconds as most OpenNMS configs are in milliseconds, but in this case that would remove granularity from the user's control. If we could somehow make it obvious to the user that this option is in nanoseconds, I would lean toward giving them this control. Not having used this feature, I'm ambivalent to the decision.

Reply:
we are parsing it and directly setting sampling frequency here. Not doing any conversions.  I would prefer to keep this way.   If some implementations doesn't stick to ns or ms, user has flexibility to update.

Reply:
make it accept N.nnn ? :D